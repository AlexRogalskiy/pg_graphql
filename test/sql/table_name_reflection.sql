begin;

    create table public.account_holder(id int);

    select gql.to_type_name('public.account_holder');

    set search_path = '';

    select gql.to_type_name('public.account_holder');

    create table public."5ac!C h&"(id int);

    select gql.to_type_name('public."5ac!C h&"');

rollback;
