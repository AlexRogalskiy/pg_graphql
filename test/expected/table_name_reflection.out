begin;
    create table public.account_holder(id int);
    select gql.to_type_name('public.account_holder');
  to_type_name  
----------------
 account_holder
(1 row)

    set search_path = '';
    select gql.to_type_name('public.account_holder');
     to_type_name      
-----------------------
 public_account_holder
(1 row)

    create table public."5ac!C h&"(id int);
    select gql.to_type_name('public."5ac!C h&"');
  to_type_name   
-----------------
 public_5ac_C_h_
(1 row)

rollback;
